rbac/management/role/definer.py

```diff
@@ -114,8 +119,12 @@ def _make_role(data, config: _SeedRolesConfig):
             if config.force_create_relationships:
                 dual_write_handler.replicate_new_system_role()
                 logger.info("Replicated system role %s", name)
-                return role
-            logger.info("No change in system role %s", name)
+            else:
+                logger.info("No change in system role %s", name)
+            # Still seed V2 role even if V1 unchanged
+            _seed_v2_role_from_v1(
```

Comment:
The Role.objects.public_tenant_only().filter(name=name).update(...) above does not update the object in-memory, maybe we need to put role=....update(...) to get a new instance. Or use refresh_from_db()